package.json

```diff
@@ -23,7 +23,7 @@
     "build": "next build",
     "dev": "next",
     "lint": "npm run lint:eslint",
-    "lint:eslint": "eslint --ignore-path .gitignore \"**/*.{jsx,js}\"",
```

Comment:
Can you revert this line? The intent here is that eslint ignores what files are listed in .gitignore